### DISCUSSION TITLE
由于超时无法访问阿里目录

### DISCUSSION THREAD

**Discussion by apang1945** [AlistGo/alist#2703] [Q&A]
想问一下这个
failed get objs: failed to list objs: Post "https://api.aliyundrive.com/v2/file/list": context deadline exceeded (Client.Timeout exceeded while awaiting headers)
是怎么了，我是国外的服务器，登录阿里云盘，然后刷新不出来
我的服务器的IP昨天被禁了，所以我给我的ip:5244走了代理，开始界面可以出来，但是一点开阿里云盘的目录就报错
这有什么解决办法吗？求大佬帮帮我或者有什么解决办法
[之前有一个兄弟是这样](https://github.com/alist-org/alist/discussions/2456)
但又和我的问题不太一样
![1](https://user-images.githubusercontent.com/80013555/207300229-bcb3e3e6-6cee-44a7-a88b-8d6ed029a4ad.jpg)

**Comment by anwen-anyi** [comment]
国外服务器连不上阿里也正常，阿里禁止国外的链接，
我开代理在  [aliyundrive.com/drive](https://www.aliyundrive.com/drive) 网页版本上传都上传不了，代理一关啥事儿也没

**Comment by 1738394450** [comment]
我的也是这样的国外vps，老哥解决了吗

**Reply by Jue-Xu** [reply]
试试vps换ip？

**Reply by apang1945** [reply]
只能换 ip 了，或者你如果只想用来看alist 的话，可以试试 warp 一键脚本，只使用 ipv4，可以正常访问，但是其他的docker 搭建的内容就莫名打不开，或者是需要科技

**Reply by Jue-Xu** [reply]
换ip一定能解决吗，是不是一定要换一个没有被墙的ip？

**Reply by apang1945** [reply]
嗯嗯，换一个没有墙的 ip，我的服务商换一次 5 刀，难受😣

**Reply by Jue-Xu** [reply]
我的Racknerd 3刀换一个，不保证换的不被墙

**Comment by 1738394450** [comment]
收到

**Reply by Jue-Xu** [reply]
好快啊，我还没试换IP呢，我感觉是我的VPS IP被墙了以后，阿里云盘连不上了。
这里有个回答 https://github.com/alist-org/alist/discussions/2207#discussioncomment-4297327 ， 不过不是VPS。
如果换IP不行的话，到时候再讨论。

**Comment by Jue-Xu** [comment]
找VPS运营商换了IP，这个问题自动解决。但是不清楚的是 之前IP被墙是不是因为VPS上跑了阿里云盘的流量，如果是这样的话，之后继续用阿里云盘还是会导致被墙，这样实用性就不高了（每次换IP还要钱）
